.github/workflows/release-azure-pipelines.yml

```diff
@@ -0,0 +1,103 @@
+# This pipeline will be triggered when either main branch is pushed or 2AM on workdays.

```

Comment:
Where is the 2am on workday stuff defined?

Reply:
Oh you know what! I copied this over from the CG pipeline, and the comment is no longer relevant. I'll remove it. Thanks for catching that!

Reply:
Thinking about it more, I'm actually going to change it to only be manually triggered (to better replicate the existing workflow).